### PR TITLE
#76: Added JDBC connection options

### LINKED ISSUE #76: Enable more jdbc properties in connection definition
User should have more flexibility while specifying the connection to exasol, in case other options for jdbc connection are needed. 

| Option | Value |
| - | - |
| spark.exasol.jdbc_options | debug=1;logdir=/tmp/my folder/;schema=sys |

### PR DESCRIPTION
**Pull request by 3cham**
This fixes #76

### CODE REVIEW

File: src/main/scala/com/exasol/spark/util/ExasolConfiguration.scala [hunk 1]
```diff
@@ -65,13 +66,35 @@ object ExasolConfiguration {
       )
   }
 
+  def checkJdbcOptions(str: String): String = {
+    if (str.endsWith(";") || str.startsWith(";")) {
+      throw new IllegalArgumentException(
+        "jdbc option should not start or end with semicolon"
```
Comment (morazow) [inline src/main/scala/com/exasol/spark/util/ExasolConfiguration.scala:72]:
```suggestion
        "Jdbc options should not start or end with semicolon."
```

File: src/main/scala/com/exasol/spark/util/ExasolConfiguration.scala [hunk 2]
```diff
@@ -65,13 +66,35 @@ object ExasolConfiguration {
...
+      )
+    }
+
+    if (str.length > 0) {
+      str
+        .split(";")
+        .foreach(kv => {
+          if (kv.filter(_ == '=').length != 1) {
+            throw new IllegalArgumentException(
+              s"property invalid: $kv does not have key=value format"
```
Comment (morazow) [inline src/main/scala/com/exasol/spark/util/ExasolConfiguration.scala:82]:
```suggestion
              s"Invalid property: $kv does not have key=value format."
```

**Review by morazow — CHANGES_REQUESTED** [review]
Hey @3cham,

Thanks for the pull request!

I have minor change requests to the exception messages, then all looks good.

**Comment by 3cham** [comment]
@morazow : that was fast 👍 yeah I fixed the exception messages. Hope you're still doing well 

Take care

**Review by morazow — APPROVED** [review]

**Comment by morazow** [comment]
Hey @3cham,

Great to hear from you!

Thanks again sending the patch! We can release quick new version soon.

Best

**Pull request merged**